### PR TITLE
Adds enums and class for data packet headers

### PR DESCRIPTION
**Pull request by Smile040501**

### CODE REVIEW

File: src/PlexShareScreenshare/Utils.cs [hunk 1]
```diff
@@ -0,0 +1,22 @@
+﻿/// <author>Mayank Singla</author>
```
Comment (tzublue) [inline src/PlexShareScreenshare/Utils.cs]: This file is probably not needed. Each class should own their own constants.

Comment (Smile040501) [reply src/PlexShareScreenshare/Utils.cs]: Removed!!!

File: src/PlexShareScreenshare/DataPacket.cs [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        /// and data field in the packet used for communication between server
+        /// and client
+        /// </summary>
+        /// <param name="header">
+        /// Header present in the packet
+        /// </param>
+        /// <param name="data">
+        /// Data present in the packet
+        /// </param>
+        public DataPacket(string header, string data)
```
Comment (tzublue) [inline src/PlexShareScreenshare/DataPacket.cs]: Make sure DataPacket can handle all sorts of packets we want to send. Maybe writing in comments how the packet will look like for different headers is a good idea.

**Review by tzublue — COMMENTED** [review]
Can merge after addressing the comment

File: src/PlexShareScreenshare/DataPacket.cs (hunk 2 above)
Comment (Smile040501) [reply src/PlexShareScreenshare/DataPacket.cs]: Done :)

**Pull request merged**